cmd/edgectl/aes_install.go

```diff
@@ -897,24 +915,45 @@ func (i *Installer) CaptureKubectl(name, input string, args ...string) (res stri
 		return
 	}
 	kargs = append([]string{kubectl}, kargs...)
-	return i.Capture(name, input, kargs...)
+	return i.Capture(name, true, input, kargs...)
+}
+
+// SilentCaptureKubectl calls kubectl and returns its stdout
+// without dumping all the output to the logger.
+func (i *Installer) SilentCaptureKubectl(name string, input string, args ...string) (res string, err error) {
+	res = ""
+	kargs, err := i.kubeinfo.GetKubectlArray(args...)
+	if err != nil {
+		err = errors.Wrapf(err, "cluster access for %s", name)
+		return
+	}
+	kubectl, err := i.GetKubectlPath()
+	if err != nil {
+		err = errors.Wrapf(err, "kubectl not found %s", name)
+		return
+	}
+	kargs = append([]string{kubectl}, kargs...)
+	return i.Capture(name, false, input, kargs...)
 }
 
 // GetKubectlPath returns the full path to the kubectl executable, or an error if not found
 func (i *Installer) GetKubectlPath() (string, error) {
 	return exec.LookPath("kubectl")
 }
 
-// Capture calls a command and returns its stdout, dumping all output to the
-// logger.
-func (i *Installer) Capture(name, input string, args ...string) (res string, err error) {
+// Capture calls a command and returns its stdout
+func (i *Installer) Capture(name string, logToStdout bool, input string, args ...string) (res string, err error) {
 	res = ""
 	resAsBytes := &bytes.Buffer{}
 	i.log.Printf("$ %s", strings.Join(args, " "))
 	cmd := exec.Command(args[0], args[1:]...)
 	cmd.Stdin = strings.NewReader(input)
-	cmd.Stdout = io.MultiWriter(NewLoggingWriter(i.cmdOut), resAsBytes)
-	cmd.Stderr = NewLoggingWriter(i.cmdErr)
+	if logToStdout {
+		cmd.Stdout = io.MultiWriter(NewLoggingWriter(i.cmdOut), resAsBytes)
+		cmd.Stderr = NewLoggingWriter(i.cmdErr)
+	} else {
+		cmd.Stdout = io.MultiWriter(resAsBytes)
+	}
```

Comment:
Why use an `io.MultiWriter` here? Just `resAsBytes` would suffice, wouldn't it? Also, what about `cmd.Stderr`?